controllers/articles.js

```diff
@@ -1,24 +1,36 @@
+const comments = require('../db/data/test-data/comments');
 const {
   fetchArticleByID,
   updateVotesOnArticleByID,
   fetchArticles,
+  fetchArticleComments,
 } = require('../models/articles');
 
+exports.getArticles = (req, res, next) => {
+  fetchArticles()
+    .then((articles) => res.status(200).send(articles))
+    .catch((err) => next(err));
+};
+
 exports.getArticlesByID = (req, res, next) => {
   fetchArticleByID(req.params.article_id)
     .then((article) => res.status(200).send(article))
     .catch((err) => next(err));
 };
 
-exports.getArticles = (req, res, next) => {
-  fetchArticles()
-    .then((articles) => res.status(200).send(articles))
-    .catch((err) => next(err));
-};
 exports.patchArticleByID = (req, res, next) => {
   const { article_id: id } = req.params;
   const { inc_votes: vote } = req.body;
   updateVotesOnArticleByID(id, vote)
     .then((article) => res.status(200).send(article))
     .catch((err) => next(err));
 };
+
+exports.getArticleComments = (req, res, next) => {
+  const { article_id: id } = req.params;
+  fetchArticleComments(id)
+    .then((comments) => {
+      res.status(200).send(comments);
```

Comment:
looking for `.send({comments: comments})` structure

Reply:
OH MY DAYS, 

This explains why I couldn't put body on the key of comments. I was busting my brain over this for so long!
✅